### ISSUE TITLE
Sorry. The coin/currency pair BTC/USD does not exist in the cryptocompare api.

### ISSUE THREAD

**Issue by hopitas** [madnight/bitcoin-chart-cli#23]
Did the api change somehow? The Cryptocompare site is still up and their api is working.

**Comment by madnight** [comment]
Do you still face this issue?

**Comment by hopitas** [comment]
Seems so. Has cryptocompare api changed somehow?

**Comment by madnight** [comment]
Okay have you tried? 
``` npx bitcoin-chart-cli@3.3.3 ```

**Comment by hopitas** [comment]
same result :(

**Comment by madnight** [comment]
Not sure, how I can help you atm. I do not face any problems with the current master (3.3.3) on my computer. Also in CI you can see charts get printed and tests run through: https://travis-ci.org/github/madnight/bitcoin-chart-cli/jobs/730417518

Which operating system are you on? Have you tried a different device?

**Comment by solovenkoU** [comment]
The error handling seems to be nonexistent here.
The problem likely connected with rate limitation on cryptocompare service.
One could upgrade their account and use api key. Or one could rewrite bitcoin-chart-cli to use ax proxy.

**Comment by madnight** [comment]
> The error handling seems to be nonexistent here.

Yeah, sorry for that. I improved the error handling a bit in version 3.5.5. Now it prints the error response from the API, including rate limiting.

> The problem likely connected with rate limitation on cryptocompare service.
One could upgrade their account and use api key. 

Yes, or one could try to lower the request frequency.

> Or one could rewrite bitcoin-chart-cli to use ax proxy.

Never heard of ax proxy.
